arrocke/gloss-translation
# fix: missing i18n keys

## What has changed

- In `UsersView`, the invalid translation key `users:name`, used in the users table header, was changed to `common:name`
- In `ManageLanguageView`, the invalid translation key `users:roles`, used in the language members table header, was changed to `users:role` with a `count` of 100.
  - Also, the text "Remove" was internationalized with the new translation key `common:remove`
## Connected Issues
closes #345 


## QA Steps

1. Change the interface language to Arabic (so un-internationalized text becomes more obvious)
2. Navigate to /users
3. Confirm that all the table headers are displayed in Arabic
4. Navigate to /languages/eng
5. Confirm that all the table headers are displayed in Arabic. 
6. Confirm that the "Remove" text button in each language member entry is displayed in Arabic.

## Post-Deployment
